client/js/components/user/DpUserList/DpUserFormFields.vue

```diff
@@ -271,7 +271,7 @@ export default {
     },
 
     isManagingSingleOrganisation () {
-      return hasPermission('area_organisations') === false && this.presetUserOrgaId
+      return hasPermission('feature_organisation_user_list') === false && this.presetUserOrgaId
```

Comment:
```suggestion
      return !hasPermission('feature_organisation_user_list') && this.presetUserOrgaId
```

Reply:
done in [41b9b0c](https://github.com/demos-europe/demosplan-core/pull/3475/commits/41b9b0cc616c8556a288e1c5d05e8b4613494e1b)